app/views/carts/index.html.erb

```diff
@@ -33,7 +33,7 @@
 	  <thead>
 	    <tr>
 	      <th>Name</th>
-        <th>Image</th>
+        <th></th>
```

Comment:
good call, we don't need the header to display image, people should be able to tell that it's an image!